br/pkg/lightning/backend/tidb/tidb.go

```diff
@@ -870,7 +870,7 @@ func (*Writer) IsSynced() bool {
 // TableAutoIDInfo is the auto id information of a table.
 type TableAutoIDInfo struct {
 	Column string
-	NextID int64
+	NextID string
```

Comment:
it should be uint64. But seems no one is using this member? We can directly delete it.

Reply:
https://github.com/pingcap/tidb/blob/ac0c90e232436573e1765ba944011eea028b9996/br/pkg/lightning/backend/tidb/tidb.go#L911-L915

will use this `NextID`. So if we change it to uint64, this `append` would result in error "Cannot use 'nextID' (type string) as the type uint64"